rtv/content.py

```diff
@@ -510,3 +574,52 @@ def get(self, index, n_cols=70):
         data['offset'] = 0
 
         return data
+
+
+class MultiredditContent(Content):
+
+    def __init__(self, multireddits, loader):
+
+        self.name = "Multireddits"
+        self.order = None
+        self._loader = loader
+        self._multireddits = multireddits
+        self._multireddit_data = []
+
+        try:
+            self.get(0)
+        except IndexError:
+            raise exceptions.SubscriptionError('No multireddits')
+
+    @classmethod
+    def from_user(cls, reddit, multireddits, loader):
+        multireddits = (m for m in multireddits)
```

Comment:
Two things here:
- the praw methods `get_multireddits` and `get_my_multireddits` return lists rather than generators. I turn it into a generator to maintain some similarity in the code.

- I want this method to potentially load not just the logged in user's multireddit but also any desired user. So either list of multireddits or the username should be passed to this method.